### ISSUE TITLE
Feature request: .desc  = FALSE option for fct_infreq()

### ISSUE THREAD

**Issue by jtr13** [tidyverse/forcats#209]
For horizontal bar charts, it is better to sort by increasing order of frequency, since bars are plotted bottom to top. A `.desc = FALSE` option in `fct_infreq()` would be cleaner than using `fct_rev()`:

df <- tibble(x = c("red", "red", "red", "yellow", "yellow", "blue"))
ggplot(df, aes(fct_rev(fct_infreq(x)))) + geom_bar() + coord_flip()

![image](https://user-images.githubusercontent.com/14161912/65453831-f5bb1680-de11-11e9-93cd-6a04d2e0cc2d.png)

**Comment by jtr13** [comment]
This has been addressed in pull request #210

**Comment by hadley** [comment]
I don't think an argument is the right solution because it would result in more net typing:

```R
ggplot(df, aes(fct_rev(fct_infreq(x)))) + geom_bar() + coord_flip()
ggplot(df, aes(fct_infreq(x, .desc = TRUE))) + geom_bar() + coord_flip()
```
